### ISSUE TITLE
Question about debugger

### ISSUE THREAD

**Issue by Vertygo** [gregg-miskelly/MIEngine-wiki#1]
Hi @gregg-miskelly ,

first of all, I want to say sorry for writing this question here in the non-related to question topic ...
I saw [blog post](https://blogs.msdn.microsoft.com/greggm/2008/06/09/writing-a-script-debugger-without-using-the-mdm/) about writing script debugger without MDM and I'm wondering if its possible to write ASP classic debugger using PDM (attach to w3wp process and debug)? If not, do you have any guidelines on how it would be possible? I want to help out [this](https://github.com/Luncher91/ActivDbgNET) project to write VBScript/ASP (if possible) debugger for VS Code.

**Comment by gregg-miskelly** [comment]
Yes, using the PDM would still be the right way to create a script debugger.

One note: I think at some point my old blog is going to get deleted. So if you find that stuff useful I would suggest saving it.
